bin/install-wp-tests.sh

```diff
@@ -50,10 +50,15 @@ install_wp() {
 
   if [ $WP_VERSION == 'latest' ]; then
     local ARCHIVE_NAME='latest'
+    # fetch the latest wordpress version code
+    VERSION_CODE=$(curl -s "https://api.wordpress.org/core/version-check/1.7/" | jq -r '[.offers[]|select(.response=="upgrade")][0].version')
```

Comment:
Is `jq` really included in CI systems?

Reply:
I was wondering the same. I didn't find a good resource but found this: https://github.com/travis-ci/travis-ci/issues/7954#issuecomment-310527961.

Reply:
Seems to work when testing on a different project